moped-etl/arcgis/settings.py

```diff
@@ -3,8 +3,11 @@
 LAYER_IDS = {"points": 0, "lines": 1, "combined": 2, "exploded": 3}
 
 COMPONENTS_QUERY_BY_LAST_UPDATE_DATE = """
-query GetProjectsComponents($where: component_arcgis_online_view_bool_exp!) {
-  component_arcgis_online_view(where: $where) {
+query GetProjectsComponents($project_where: moped_project_bool_exp!, $component_where: component_arcgis_online_view_bool_exp!) {
+  moped_project(where: $project_where) {
+    project_id
```

Comment:
I thought about using a [skip directive](https://hasura.io/docs/2.0/queries/postgres/variables-aliases-fragments-directives/#skipif-boolean) here in the query since we don't need them for the full replace mode. But, adding another variable and more logic in this query felt like a lot to prevent getting these ids in rare cases when we run the full replace.